swift/ql/src/queries/Security/CWE-312/CleartextLogging.ql

```diff
@@ -0,0 +1,24 @@
+/**
+ * @name Clear-text logging of sensitive information
+ * @description Logging sensitive information without encryption or hashing can
```

Comment:
No problem with the `@description` but it explicitly mentions hashing where the text in the `.qhelp` file doesn't. It might be worth altering this slightly to be more generalised or mentioning hashing in the `.qhelp` file(?)

Reply:
```suggestion
 * @description Logging sensitive information without encryption can
```